### DISCUSSION TITLE
Gists Get no Love?

### DISCUSSION THREAD

**Discussion by Justintime50** [community/events#44] [Keynotes]
I was really hoping Gists would get some love - some announcement at the keynote? There's still no way to search or organize your gists, the UI is vastly different than the normal GitHub repo views. With multiple pages of gists, it's difficult to manage. 

Is there a place GitHub takes feature suggestions or a forum we can bring this up in to get it viewed by someone that can take it to their product team?

**Comment by ossanna16** [comment]
Hi there @Justintime50! Great questions! We have a big [Product Feedback section in the Community in Discussions](https://github.com/orgs/community/discussions/). I would encourage you to post your thoughts there.

**Reply by Justintime50** [reply]
Gists isn't even a category there 😭 Thanks!

**Reply by ossanna16** [reply]
I know 😞 I suppose it's one of the lesser used features but your feedback will still be appreciated!

**Comment by omerbensaadon** [comment]
As a PM at GitHub, I also feel Gists don't get enough love 💔 (I am one of the few PMs I know that uses them regularly)

Help me make the case! **What do better gists mean to you?** What features are missing? 

What would better Gists mean for how you (or your peers) use GitHub?

**Reply by Justintime50** [reply]
I love Gists! Great place to dump snippets or short guides.

The biggest two things missing in my mind that would make them great is `search` and `lists` like repositories or stars have. I'd love to be able to search through my gists for keywords or titles instead of clicking through a half dozen pages to find one of my first few Gists that I use often. Being able to organize them into categories or lists would also help the navigation drastically.

It also feels awkward that Gists are their own interface? Why is everything on GitHub under a unified UI while Gists get their own and you have to "go back to GitHub" to dive back into repos? Shouldn't Gists act and feel just like a list of repos?

**Comment by monperrus** [comment]
FTR, product feedback about Gists that are not integrated in Github search: https://github.com/orgs/community/discussions/138097

**Comment by xTalhaAhmedCho** [comment]
![chrome_eTjRY3DcEQ](https://github.com/user-attachments/assets/247865c8-f382-48c5-a43a-de6bd50d3803)

`Dear GitHub Support Team,`

> I am writing to request the re-enablement of GitHub Actions for my account, **@TalhaAhmedCho**. I have noticed that GitHub Actions is currently disabled for my account. 

I understand that my GitHub Actions may have been disabled due to `misunderstanding of limits`. If my actions inadvertently violated any GitHub terms or policies, I sincerely apologize. It was not my intention to misuse the platform. I am committed to using GitHub Actions responsibly and in accordance with all guidelines in the future.

> I value GitHub's services greatly, especially GitHub Actions, which is an **_integral part of my workflow_** for personal project. I rely on GitHub Actions for `learning new technologies. `

**_I kindly request you to review my account and re-enable GitHub Actions. I am willing to cooperate fully and take any necessary steps to ensure I comply with GitHub's policies and guidelines in the future._**

`My GitHub username is:`  **@TalhaAhmedCho**

Thank you for your **_time and consideration_**. I look forward to your positive response and the re-enablement of GitHub Actions.

Sincerely,

Talha Ahmed Chowdhury
[Linkedin](https://www.linkedin.com/in/TalhaAhmedCho/) | [Facebook](https://www.facebook.com/@TalhaAhmedCho) | [YouTube](https://www.youtube.com/@TalhaAhmedCho) | [Twitter](https://x.com/_TalhaAhmedCho)
